src/cata_tiles.cpp

```diff
@@ -1125,7 +1125,7 @@ bool cata_tiles::draw_from_id_string(std::string id, TILE_CATEGORY category,
     }
 
     if (it == tile_ids.end()) {
-        long sym = -1;
+        wchar_t sym = -1;
```

Comment:
If it's not a matter of "will it work", it's still a matter of not behaving as expected, on some systems sym is initialized to -1, on some it's initialized to 65535, and on others it's initialized to 4294967295, that's asking for trouble :)